controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -852,4 +850,28 @@ export class OrganizationRepository {
         ),
       );
   }
+
+  public async addOidcProvider(input: { name: string; organizationId: string; endpoint: string }) {
+    await this.db
+      .insert(oidcProviders)
+      .values({ name: input.name, organizationId: input.organizationId, endpoint: input.endpoint })
+      .execute();
+  }
+
+  public async getOidcProvider(input: { organizationId: string }) {
```

Comment:
Please put this in a separate repository.

Reply:
It is organization based right?

Reply:
Yes

Reply:
then why a separate repository

Reply:
You need plenty of methods to handle OIDC stuff. I find it natural to put this into a different repository because they aren't built on top of the same tables. Repositories are should unify the access to the underlying storage. Services combine multiple repositories and services to realize high-level requirements.

Reply:
Done